### ISSUE TITLE
Object Detection (SSD-VGG): Loss nan 

### ISSUE THREAD

**Issue by hoangcuongbk80** [dusty-nv/jetson-inference#662]
Hi,

I trained the SSD for object detection using VGG instead of MobileNet and get loss nan as below:

![2](https://user-images.githubusercontent.com/10355081/88891938-4d04f100-d244-11ea-803f-a43f13565a04.png)

Pre-trained model VGG was downloaded from [here](https://github.com/qfgaohao/pytorch-ssd) .I trained my own data on Jetson Xavier. It's ok if training using MobileNet as in [instruction](https://github.com/dusty-nv/jetson-inference/blob/master/docs/pytorch-ssd.md). Any suggestion to make it work with VGG?

Thanks!

**Comment by dusty-nv** [comment]
Hi @hoangcuongbk80, I have not tested SSD-VGG - you might want to check the upstream repo here:  https://github.com/qfgaohao/pytorch-ssd

**Comment by hoangcuongbk80** [comment]
Thank you!
